classes/helper/front/form.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * NOVIUS OS - Web OS for digital communication
+ *
+ * @copyright  2011 Novius
+ * @license    GNU Affero General Public License v3 or (at your option) any later version
+ *             http://www.gnu.org/licenses/agpl-3.0.html
+ * @link http://www.novius-os.org
+ */
+
+namespace Nos\Form;
+
+class Helper_Front_Form
+{
+    /**
+     * Gets the class name for the specified width
+     *
+     * @param $width
+     * @return mixed
+     */
+    public static function getWidthClassName($width)
+    {
+        $widths = array(
```

Comment:
You could also do that (with the right context of course, or maybe with `getParsleyLocale`) : 
```php
$nf = new \NumberFormatter('en_GB', \NumberFormatter::SPELLOUT);
echo $nf->format($width);
```